### DISCUSSION TITLE
CollectionBuilder GH in spanish?

### DISCUSSION THREAD

**Discussion by gustavolsj** [CollectionBuilder/collectionbuilder.github.io#120] [Q&A] [answered]
Found this repo https://github.com/owikle/cb-multilingual-again and I really like the translatation work done on the interface.  Is there any way to use it as my template?

Is there any other way to have CB in spanish?

Thanks for your great work!

G.

**Comment by owikle** [comment, marked as answer]
Hi @gustavolsj thanks for reaching out. I went ahead and made https://github.com/owikle/cb-multilingual-again a template so you can copy it if you'd like. However, a few caveats: there's no documentation for it yet, and it's using an older version of CB-GH so some things will look and function a little differently than sites built with the current CB-GH. You can get an idea of how it's working by looking at the metadata csv and the config-translate.csv (both in the _data folder). I'm working with some partners at the University of Houston on an updated multilingual version of the current CB-GH: https://github.com/gbventura/pinatabooks, with hopes to turn the code from that project into a template soon.

You also might take a look at our cb-translate repository: https://github.com/CollectionBuilder/cb-translate, a prototype with language options for CB-CSV. Once complete it would allow CB-CSV users to build out their sites in a language of their choosing (but not toggle between languages the way that the multilingual template does). The cb-translate repo is under active development so you may encounter some bugs if you try it out, but there is a little documentation if you're curious: https://github.com/CollectionBuilder/cb-translate/blob/main/docs/localization.md. Spanish is one of the languages that's already been added in as an option.

Hopefully that helps! Good luck and let us know if you have other questions.

**Comment by gustavolsj** [comment]
Thank you very much @owikle for your detailed answer, for the time being I will stick to https://github.com/owikle/cb-multilingual-again and I will be sure to take a look at the data folder to see how it works. 
Best,

**Reply by evanwill** [reply]
@gustavolsj I just wanted to add that [cb-translate](https://github.com/CollectionBuilder/cb-translate) is fully functional matching current cb-csv right now. It translates the full UI simply by changing the config (for english, spanish, or german at the moment), but like @owikle mentioned does not switch between languages on the live site (you just build out one language). So if you want an all Spanish version, it might be your best bet--if you use it please send along feedback so we can develop it further (or tweaks to the translations, etc)!

**Reply by gustavolsj** [reply]
Hi Evan, Thanks for the info, I will be teaching a workshop/demonstration and I think for now the GH version is best for a begginners class dynamic.  Hopefully later we can make the jump to the CSV version once we learn about installing and using Ruby & Jekyll.

Cheers
